packages/world/mud.config.ts

```diff
@@ -8,7 +8,7 @@ export default defineWorld({
     generateSystemLibraries: true,
   },
   enums: {
-    Action: [
+    NotificationType: [
```

Comment:
Does renaming this require any migration? or no cuz the enum is just a uint8?

also the field is still action on the table so maybe we should keep it this?

Reply:
it shouldn't require a migration, we can just rename in the table. I don't like action, like Death is not an action

Reply:
we can also deprecate this table for a new one where the field isnt called action, up to you though